starter-code/main.js

```diff
@@ -53,19 +54,200 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
+              }
+            })
+            break;
+
+          case "10":
+            // 10.- List the name of all the products of Facebook
+            db.collection('companies').find({ "name": "Facebook" }, { name: 1, 'products.name': 1, _id: 0 }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result[0]);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
 
 
+          case "11":
+            // 11.- List the people that are working at Facebook right now (check relationships field)
+            db.collection('companies').find({ $and: [{ "name": "Facebook" }, { "relationships.is_past": false }] }, { 'relationships.person.first_name': 1, 'relationships.person.last_name': 1, _id: 0 }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+
+
+          case "12":
+            //12.- List all the companies where "david-ebersman" has worked.
+            db.collection('companies').find({ $and: [{ "relationships.person.first_name": "David" }, { "relationships.person.last_name": "Ebersman" }] }, { name: 1, _id: 0 }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "13":
+            // 13.- List by name the competitors of Facebook
+            db.collection('companies').find({ "name": "Facebook" }, { 'competitions.competitor.name': 1, _id: 0 }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "14":
+            // 14.- Names of the companies that has "social-networking" in tag-list (be aware that the value of field is a string check regex operators)
+            db.collection('companies').find({ "tag_list": { $in: ["social-networking"] } }, { name: 1, _id: 0 }).toArray((error, result) => {
```

Comment:
Si te fijas el enunciado menciona los regex operators. 

Si cuando escribes un string en lugar de hacerlo así: "social-networking" escribes /social-networking/ lo que creas es una expresión regular. Es un tipo de objeto capaz de buscar coincidencias parciales de texto. 

Usando eso se puede hacer:

```suggestion
            db.collection('companies').find({ "tag_list": /social-networking/ }, { name: 1, _id: 0 }).toArray((error, result) => {
```